content/zh_tw.html

```diff
@@ -386,6 +387,98 @@
 
 <article>
 
+<h1><a name="expect">語法 expect 與 should</a></h1>
+
+<!-- <p>
+On new projects always use the <code>expect</code> syntax.
+</p> -->
+
+<p>
+在 RSpec3 總是使用 <code>expect</code> 語法。
```

Comment:
I agree it is clearer. But you should change English first, then make your improvement in zh_tw version. Otherwise, this is not a correct translation. Also, how about when RSpec 4 came out? Will you update this text? English description is agnostic to RSpec version changes.